src/BrightScriptDebugSession.ts

```diff
@@ -826,7 +865,7 @@ interface LaunchRequestArguments extends DebugProtocol.LaunchRequestArguments {
      * so that breakpoints can be set in the source files when debugging. In order for this to work, your build process cannot change
      * line offsets between source files and built files, otherwise debugger lines will be out of sync.
      */
-    debugRootDir: string;
+    sourceDirs: string[];
```

Comment:
We still need to have `debugRootDir` in this interface for now, until it actually gets removed, so be sure to add that back in, and add some type of @deprecated annotation or message on the comment.